dev-docs/asb/README.md

```diff
@@ -111,7 +111,9 @@ For market making the ASB offers the following parameters in the config:
 min_buy_btc = 0.0001
 max_buy_btc = 0.0001
 ask_spread = 0.02
-price_ticker_ws_url = "wss://ws.kraken.com"
+price_ticker_ws_url_kraken = "wss://ws.kraken.com"
+price_ticker_ws_url_bitfinex = "wss://api-pub.bitfinex.com/ws/2"
+price_ticker_rest_url_kucoin = "https://api.kucoin.com/api/v1/bullet-public"
```

Comment:
the `v1/bullet-public` and `ws/2` suffixes are arguably part of the protocol and should not be configurable? not sure though, I'd also be fine with leaving this as is